base.yml

```diff
@@ -20,6 +20,9 @@ services:
       POSTGRES_PASSWORD: "m@0dl3ing"
       POSTGRES_DB: moodle
   selenium:
-    image: selenium/standalone-firefox:2.53.1
+    image: selenium/standalone-firefox-debug:2.53.1
+    build: ./selenium/
```

Comment:
I think that image and build are conflicting here? If this is working I assume that build is winning as the last one.

Reply:
So we just need build and not image since the Dockerfile in build is already there?

Reply:
Yep, the image is not being used, you either use an image or build.

But for the reasons I mention below, I don’t think you need to build anyway.